### PR TITLE
refactor: UPDATE TO SECOND VERSION OF COVE LIBS

### PR DESCRIPTION
**Pull request by odscjames**
Adds message Queue
Adds sample mode

https://github.com/openownership/cove-bods/issues/79

### CODE REVIEW

**Comment by odscjames** [comment]
When this is put live, must run:

    dokku ps:scale XXX  web=1 worker=1

**Comment by odscjames** [comment]
Also need one-off migration script for existing cove content

File: cove_bods/views.py [hunk 1]
```diff
@@ -1,134 +1,85 @@
...
+
+    forms = {
+        "json": {
+            form_name: form_class()
+            for form_name, form_class in JSON_FORM_CLASSES.items()
+        },
+    }
+
+    return render(request, "cove_bods/index.html", {"forms": forms})
+# cove_web_input_error(func): ???????????????
```
Comment (Bjwebb) [inline cove_bods/views.py:71]: Should this still be here?

File: cove_bods/process.py [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+            if len(file_types_reduced) == 1:
+                self.supplied_data.format = self.map_file_type_to_format[file_types_reduced[0]]
+                self.supplied_data.save()
+
+            elif len(file_types_reduced) == 0:
+                raise Exception("GOT ZERO")
+                # TODO
+
+            elif len(file_types_reduced) > 1:
+                raise Exception("GOT MORE THAN ONE")
```
Comment (Bjwebb) [inline cove_bods/process.py:67]: `NotImplementedError` would be a good choice here.

File: cove_bods/process.py [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+            context["download_xlsx_size"] = os.stat(self.xlsx_filename).st_size
+        else:
+            context["can_download_xlsx"] = False
+        # done!
+        return context
+
+
+class PythonValidateTask(TaskWithState):
+
+    state_filename: str = "python_validate.py"
```
Comment (Bjwebb) [inline cove_bods/process.py:302]: I'm confused as to why this has a `.py` extension. Isn't it going to have json written to it?

File: cove_bods/views.py (hunk 1 above)
Comment (odscjames) [reply cove_bods/views.py]: Gone

File: cove_bods/process.py (hunk 2 above)
Comment (odscjames) [reply cove_bods/process.py]: Done

File: cove_bods/process.py (hunk 3 above)
Comment (odscjames) [reply cove_bods/process.py]: Whoops

**Comment by odscjames** [comment]
Requested changes made

**Pull request merged**